student/serializers.py

```diff
@@ -43,6 +43,7 @@ class StudentSerializer(serializers.ModelSerializer):
     LoginToken = serializers.CharField(source='get_token')
     LoginHash = serializers.CharField(source='get_hash')
     timeAcceptedTos = serializers.DateTimeField(source='time_accepted_tos', format='iso-8601')
+    favoriteNum = serializers.IntegerField(source='user.favorite_num')
```

Comment:
This may work but this should be a list of fields below where you can simply add 'favorite_num' and it should serialize accordingly

Reply:
Ok I think I did it in the most recent commit.